cil-parser/src/main/java/com/vztekoverflow/cil/parser/cli/table/CLITableClassesGenerator.java

```diff
@@ -266,6 +267,7 @@ public static void generateGetColumn(HeapPenalties penalties, String fieldName,
                         shift = 5;
                     }
 
+                    writer.println(String.format("\t\tif((isSmall && (codedValue & 0xffff) == 0xffff) || (!isSmall && (codedValue & 0xffffffff) == 0xffffffff)) return null;"));
```

Comment:
Tohle zanese dost warningů, protože to může způsobit null-excpetiony. Nevím jestli je to nejlepší způsob jak to vyřešit,. ale null je potřeba.

Šlo by přidat wrapper nebo helper metodu, která by null checknula a ta se volala jen tam kde je potřeba. Pak by se ale muselo opravit mapování na CLITable typy.

Reply:
CLITablePtr nema nejakou reprezentaci null ? Pochopil jsem spravne ze nekde muze byt hodnota v metadatech null(nemiri na dalsi tabulku...) ?

Reply:
Presne tak, treba i string heap pointery muzou bejt null. Nebo null v `TypeRef.ResolutionScope` table ma specialni vyznam.

Reply:
A co vracet nejaky singleton CLITablePtr predstavujici null. A kdyz ho budes chtit pouzit k referencovani neceho, tak to zarve(takze nejaka metoda CLITablePtr.IsNull()). Ale tak jak to je ted mne osobne nevadi, protoze je to porad prototyp...